cmd/fleetctl/convert_test.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+func TestConvertFileOutput(t *testing.T) {
+	// setup the cli and the convert command
+	app := cli.NewApp()
+	app.Commands = []*cli.Command{convertCommand()}
+	app.Reader = os.Stdin
+	app.Writer = os.Stdout
+	app.Setup()
+
+	// read the expected output file
+	expected, err := ioutil.ReadFile("testdata/convert_output.yml")
+	require.NoError(t, err)
+
+	// setup a file for the convert command to write to
+	file, err := ioutil.TempFile(os.TempDir(), "convert_output.yml")
+	require.NoError(t, err)
```

Comment:
Please use `t.TempDir()` as in https://github.com/fleetdm/fleet/blob/989e638cc04962edbba35bef7940ddd5d35bb839/server/logging/filesystem_test.go#L19. I'd like to see if using this can be encoded in a lint check (https://github.com/fleetdm/fleet/issues/1673)